cmd/eelbot/config.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/emseers/eelbot"
+	"gopkg.in/ini.v1"
+)
+
+const (
+	sectionGeneral  = "General"
+	sectionJokes    = "Jokes"
+	sectionDatabase = "Database"
+
+	keyMsgTimeout         = "msg_timeout"
+	keyMultiLineJokeDelay = "multi_line_joke_delay"
+	keyDbName             = "db_name"
+)
+
+func parseConfig(filename string) (opts *eelbot.Options, err error) {
+	var file *ini.File
+	if file, err = ini.Load(filename); err != nil {
```

Comment:
Shouldn't you explicitly ignore error by naming it ```_``` if you aren't going to handle it.

Reply:
It is checked. If you notice, the return variable is named. This function would assign to it, and if its not nil, then the function returns early with the error.